xonsh/tools.py

```diff
@@ -555,6 +555,21 @@ def ensure_int_or_slice(x):
         return int(x)
 
 
+def is_completions_display_value(x):
+    return x in {'none', 'single', 'multi'}
+
+
+def to_completions_display_value(x):
+    if str(x).lower() in {'none', 'false'}:
+        return 'none'
+    if str(x).lower() in {'multi', 'true'}:
+        return 'multi'
+    if str(x).lower() in {'single'}:
```

Comment:
> Did I do anything wrong there?

Nope that is the expected behaviour. The benefits of warnings is that users can modify how they want to see them globally through Python itself.